nncf/experimental/post_training/algorithms/quantization/min_max_quantization.py

```diff
@@ -52,27 +52,43 @@ class MinMaxQuantizationParameters(AlgorithmParameters):
     """
 
     def __init__(self,
-                 weight_quantizer_config: QuantizerConfig = None,
-                 activation_quantizer_config: QuantizerConfig = None,
+                 preset: Preset = Preset.MIXED,
+                 weight_bits: int = 8,
+                 weight_granularity: Granularity = Granularity.PERCHANNEL,
+                 activation_bits: int = 8,
+                 activation_granularity: Granularity = Granularity.PERTENSOR,
+                 range_type: RangeType = RangeType.MEAN_MINMAX,
                  number_samples: int = 100,
                  target_device: HWConfigType = HWConfigType.CPU,
-                 range_type: RangeType = RangeType.MEAN_MINMAX,
-                 quatize_outputs: bool = False,
-                 ignored_scopes: List[str] = None,
+                 quantize_outputs: bool = False,
+                 ignored_scopes: List[str] = None
                  ):
-        self.weight_quantizer_config = weight_quantizer_config
-        self.activation_quantizer_config = activation_quantizer_config
+        weight_mode, activation_mode = self._determine_weight_activation_modes(preset)
+        self.weight_quantizer_config = self._determine_quantizer_config(weight_bits, weight_granularity, weight_mode)
+        self.activation_quantizer_config = self._determine_quantizer_config(activation_bits, activation_granularity,
+                                                                            activation_mode)
         self.number_samples = number_samples
         self.target_device = target_device
         self.range_type = range_type
+        self.quantize_outputs = quantize_outputs
         self.ignored_scopes = [] if ignored_scopes is None else ignored_scopes
-        self.quantize_outputs = quatize_outputs
 
     def to_json(self) -> Dict[str, Union[str, float, int]]:
         """
         Serialize all MinMaxQuantization parameters to JSON.
         """
 
+    def _determine_weight_activation_modes(self, preset: Preset):
+        # TODO(kshpv): add support of presets
```

Comment:
Why shouldn't do it right now instead of adding TODO's?

Reply:
I've created a ticket on that - 93174

Reply:
The presets were added.